torch/testing/_internal/common_utils.py

```diff
@@ -2134,14 +2137,13 @@ def _run_with_retry(self, result=None, num_runs_left=0, report_only=True, num_re
 
         if TEST_WITH_TORCHDYNAMO:
             # TorchDynamo optimize annotation
-            if TEST_WITH_TORCHINDUCTOR:
-                super_run = torch._dynamo.optimize("inductor")(super().run)
-            else:
-                super_run = torch._dynamo.optimize("eager")(super().run)
+            super_run = torch._dynamo.optimize("eager")(super().run)
             super_run(result=result)
-
             # TODO - Reset for each test slows down testing significantly.
             # torch._dynamo.reset()
+        elif TEST_WITH_TORCHINDUCTOR:
+            super_run = torch._dynamo.optimize("inductor")(super().run)
+            super_run(result=result)
```

Comment:
Nit: Move `super_run(result=result)` out of the if-else as it runs for all branches anyway?